dhis-2/dhis-api/src/main/java/org/hisp/dhis/sqlview/SqlView.java

```diff
@@ -87,7 +87,7 @@
 
     private String sqlQuery;
 
-    private SqlViewType type;
+    private SqlViewType type = SqlViewType.VIEW;
```

Comment:
I wonder if default should also be set in the `setAutoFields` in case the fields are `null` as one could still set a `null` value.

Reply:
We have `SchemaValidation` in the importer, so if user post a `null` value, it will failed with `ErrorCode.E4000`